src/utils/server-requests.js

```diff
@@ -67,11 +67,16 @@ export const login = async (
 
     setIsLoading(false);
     navigate("/");
+    setToastMessage({ type: "success", message: "Login Successful!" });
   } catch (error) {
+    console.log(error.response);
     if (error.response?.status == 404) {
-      setLoginError("You are not registered with us. Please sign up");
+      setToastMessage({
```

Comment:
suggestion: would be good if name is something like showToastMessage instead of set